### PR TITLE
[feat]: Make search panel more compact, button more visible

### PR DESCRIPTION
**Pull request by meenal06**
Signed-off-by: Meenal Trivedi <meenaltrivedi6102@gmail.com>

## Which problem is this PR solving?
- fixes https://github.com/jaegertracing/jaeger-ui/issues/683

## Short description of the changes
- Amended the layout and design of search box according to the requested samples
- Couldn't update the upload dialogue due to limitations by dragger antd component, see here https://github.com/jaegertracing/jaeger-ui/issues/683#issuecomment-812859573

## Screenshot 
Screenshot 2021-04-07 at 1 04 22 AM

### CODE REVIEW

**Comment by meenal06** [comment]
Hi @jpkrohling @albertteoh please review

**Comment by jpkrohling** [comment]
@rubenvp8510 would you be open to review this one?

**Comment by meenal06** [comment]
Hi @rubenvp8510 I have removed the code comments

File: packages/jaeger-ui/src/components/SearchTracePage/SearchForm.js [hunk 1]
```diff
@@ -406,36 +413,49 @@ export class SearchFormImpl extends React.PureComponent {
...
+              <Field
+                name="minDuration"
+                component={ValidatedAdaptedInput}
+                placeholder={placeholderDurationFields}
+                props={{ disabled }}
+                validate={validateDurationFields}
+              />
+            </FormItem>
+          </Col>
+        </Row>
```
Comment (yurishkuro) [inline packages/jaeger-ui/src/components/SearchTracePage/SearchForm.js:458]: I think there was also an ask to make the Search button into different color.

**Comment by rubenvp8510** [comment]
This LGTM, we only need to check Yuri comment to see if the search button color will be changed.

**Comment by meenal06** [comment]
Hi @rubenvp8510 @yurishkuro , I have made the button design changes to : color change with position right

File: packages/jaeger-ui/src/components/SearchTracePage/SearchForm.js (hunk 1 above)
Comment (meenal06) [reply packages/jaeger-ui/src/components/SearchTracePage/SearchForm.js:458]: done 😄

**Comment by yurishkuro** [comment]
please add a screenshot to the description

**Comment by meenal06** [comment]
Hi @yurishkuro , I have added the screenshot for my change in the pr-description. Please for review!

**Review by yurishkuro — APPROVED** [review]

**Comment by yurishkuro** [comment]
Thanks!

**Pull request merged**